plugins/service-monitor-activemq/src/main/java/com/thinkbiganalytics/service/activemq/config/ActivemqServiceCheckSpringConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package com.thinkbiganalytics.service.activemq.config;
+
+
+/*-
+ * #%L
+ * thinkbig-service-monitor-nifi
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
the header here is incorrect and needs to be regenerated.  copy/paste error.  You can remove the header manually, and run `mvn license:update-file-header` to regenerate